src/Mvc/src/Microsoft.AspNetCore.Mvc.Core/Resources.resx

```diff
@@ -493,4 +493,12 @@
   <data name="ReferenceToNewtonsoftJsonRequired" xml:space="preserve">
     <value>'{0}' requires a reference to '{1}'. Configure your application by adding a reference to the '{1}' package and calling '{2}.{3}' inside the call to '{4}' in the application startup code.</value>
   </data>
+  <data name="ModelBinding_ExceededMaxModelBindingCollectionSize" xml:space="preserve">
+    <value>Collection bound to '{0}' exceeded {1}.{2} ({3}). Remove properties that bind unconditionally from {4}.</value>
```

Comment:
I think this is a little more terse than is helpful. In particular `Remove properties that bind unconditionally from {4}.` - I don't think that's going to be very clear to anyone that see this error. I'd suggest something like:

> This is an internal limit that is intended to prevent unbounded growth with an problematic model binder or model type. This error can be cause by writing a model like ....